iothub/device/src/Authentication/DeviceAuthenticationWithRegistrySymmetricKey.cs

```diff
@@ -12,18 +12,56 @@ namespace Microsoft.Azure.Devices.Client
     /// </summary>
     public sealed class DeviceAuthenticationWithRegistrySymmetricKey : IAuthenticationMethod
     {
+        private const int DefaultSasRenewalBufferPercentage = 15;
+        private static readonly TimeSpan s_defaultSasTimeToLive = TimeSpan.FromHours(1);
+
+        private readonly TimeSpan _suggestedTimeToLive;
+        private readonly int _timeBufferPercentage;
+
         private string _deviceId;
         private byte[] _key;
 
         /// <summary>
         /// Initializes a new instance of the class.
         /// </summary>
-        /// <param name="deviceId">Device identifier.</param>
+        /// <param name="deviceId">Device Identifier.</param>
```

Comment:
Why capitalize identifier? Is it a property noun?

Reply:
This looks like an unintentional copy-paste issue. I'll update it.